topics/admin/tutorials/external-auth/tutorial.md

```diff
@@ -44,7 +44,7 @@ For this exercise we will use a basic password file method for authenticating -
 > 1. Edit your `galaxyservers` group variables file and update the main location block defined for serving galaxy. Add the parameters:
```

Comment:
@hexylena should this happen in the galaxy.j2 template ?

Reply:
yes, it should. a previous version used a different nginx role where the config was embedded

Reply:
Cool, thanks for confirminig